### PR TITLE
Update statuses and table order

### PR DESCRIPTION
**Pull request by degray**
Add other status page status mappings and ensure table frames are always in order with a single frame.

### CODE REVIEW

File: pkg/internal/api_helpers.go [hunk 1]
```diff
@@ -165,12 +168,30 @@ func (spc *StatusPageComponentChange) getLabels() map[string]string {
...
+		"down":                 2,
+		"Disruption":           2,
+		"Critical":             2,
+		"outage":               2,
+		"degraded_performance": 2,
+		"major_outage":         2,
+		"partial_outage":       2,
+		"under_maintenance":    3,
 	}
 	result := statuses[status]
```
Comment (fmterrorf) [inline pkg/internal/api_helpers.go:196]: Should we set a default value incase a new status is added

File: pkg/plugin/queries.go [hunk 2]
```diff
@@ -35,18 +35,27 @@ const (
...
+		// For table Wide frames, we always want to append to the one single frame in order
+		if frameType == TableFrameType && frameToAppendTo == nil {
 			frameToAppendTo = &frameDefinition
-			frameMap[key] = frameToAppendTo
+			frameMap["fixed-table"] = frameToAppendTo
+		} else if frameType == GraphFrameType {
+			key := frameDataItem.GetKey()
+
+			var ok bool
+			frameToAppendTo, ok = frameMap[key]
```
Comment (fmterrorf) [inline pkg/plugin/queries.go:54]: You can probably drop the var
```suggestion
			frameToAppendTo, ok := frameMap[key]
```

Comment (degray) [reply pkg/plugin/queries.go:54]: Can't, frameToAppend to has to be declared outside.

File: pkg/internal/api_helpers.go (hunk 1 above)
Comment (degray) [reply pkg/internal/api_helpers.go:196]: I believe since this is a value type it will default to 0 in that case

**Review by fmterrorf — APPROVED** [review]

**Pull request merged**